src/elements/content-explorer/ContentExplorer.tsx

```diff
@@ -1639,13 +1639,17 @@ class ContentExplorer extends Component<ContentExplorerProps, State> {
             return;
         }
 
-        this.api.getFolderAPI(false).getFolderFields(
+        this.api.getFolderAPI().getFolder(
```

Comment:
it feels weird to call `getFolder` when we only want the folder properties and not the items. why not add the `forceFetch` functionality to `getFolderFields`?

Reply:
then we can do something like:
```
forceFetch: view === 'metadata'
```
so that it's only force fetching for metadata